---
title: "Reduced model complexity enhances QTL detection power"
author:
  - Frederick Boehm:
      email: frederick.boehm@gmail.com
      institute: [UofO]
      correspondence: true
  - Yourfirstname Yourlastname:
      email: fl@another.edu
      institute: [UofA]
      correspondence: false
institute:
  - UofO: University of Massachusetts Medical School
  - UofA: University of Another Place
date: "`r format(Sys.time(), '%d %B, %Y')`"
output: 
    bookdown::word_document2:
      fig_caption: yes
      reference_docx: "../templates/template.docx" # Insert path for the DOCX file
      pandoc_args:
      - --lua-filter=../templates/scholarly-metadata.lua
      - --lua-filter=../templates/author-info-blocks.lua
      - --lua-filter=../templates/pagebreak.lua
bibliography: references.bib
csl: "../templates/journal-of-archaeological-science.csl" # Insert path for the bib-style
abstract: |
  Multiparental populations of mice, flies, and other model organisms incorporate more genetic diversity than traditional crosses and enable high-resolution QTL mapping. However, some QTL mapping experiments have yielded modest results, possibly due to limited statistical power to detect QTL. In multiparental populations, one typically infers, for each study subject, the founder from which each haplotype across the genome was inherited. One then performs traditional QTL mapping while allowing for distinct effects of all founder alleles. In practice, the allowance for distinct allele effects reduces power when, in fact, multiple founders share a single allele. We propose to first identify the allelic series of a known QTL, then use a corresponding reduced statistical model to detect QTL in a second 
keywords: |
  quantitative trait locus; allelic series; keyword 3
highlights: |
  These are the highlights. 
---


<!-- This is the format for text comments that will be ignored during renderings. Do not put R code in these comments because it will not be ignored. -->

```{r setup, echo = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  warning = FALSE,
  message = FALSE,
  echo = FALSE,
  comment = "#>",
  fig.path = "../figures/",
  dpi = 300
)
```

# Introduction


# Background

# Methods

# Results

```{r get-data, eval = FALSE}
# Note the path that we need to use to access our data files when rendering this document
my_data <- read.csv(here::here('analysis/data/raw_data/my_csv_file.csv'))
```

```{r demo-plot, fig.cap="A plot of random numbers"}
plot(rnorm(10))
```

Figure \@ref(fig:demo-plot) shows how we can have a caption and cross-reference for a plot

```{r demo-inline-code}
x <- round(pi, 2)
```

Here is an example of inline code `r x` in the middle of a sentence. 

# Discussion

# Conclusion

# Acknowledgements

<!-- The following line inserts a page break  -->
\newpage

# References 
<!-- The following line ensures the references appear here for the MS Word or HTML output files, rather than right at the end of the document (this will not work for PDF files):  -->
<div id="refs"></div>

\newpage

### Colophon

This report was generated on `r Sys.time()` using the following computational environment and dependencies: 

```{r colophon, cache = FALSE}
# which R packages and versions?
if ("devtools" %in% installed.packages()) devtools::session_info()
```

The current Git commit details are:

```{r}
# what commit is this file at? 
if ("git2r" %in% installed.packages() & git2r::in_repository(path = ".")) git2r::repository(here::here())  
```
